larq/quantizers_test.py

```diff
@@ -325,19 +345,39 @@ def test_magnitude_aware_sign_grad(self):
             grad.numpy(), np.where(abs(a) < 1, np.ones(a.shape) * scale_vector, 0)
         )
 
-    def test_dorefa_ste_grad(self):
+    @pytest.mark.parametrize("mode", ["activations", "weights"])
+    def test_dorefa_ste_grad(self, mode):
         @np.vectorize
         def ste_grad(x):
             if x <= 1 and x >= 0:
                 return 1.0
             return 0.0
 
+        # For other tests, the golden gradient is defined using python
+        # expressions and "converted" to a numpy function using np.vectorize.
+        # But the weight quantizer does a max-operation over the full
+        # quantized tensor, so np.vectorize, which causes the gradient to be
+        # called element-wise, needs to be skipped!
+        # @np.vectorize
```

Comment:
For the other gradient test we only use `np.vectorize`, because we require an `if/else` statement which is easier to read that way. Since this is not the case here, the native numpy usage is actually preferred. So we can remove this comment.
```suggestion
```